code/go/0chain.net/chaincore/chain/config.go

```diff
@@ -520,6 +520,9 @@ func (c *ConfigImpl) FromViper() error {
 	conf.DbsEvents.MaxIdleConns = viper.GetInt("server_chain.dbs.events.max_idle_conns")
 	conf.DbsEvents.MaxOpenConns = viper.GetInt("server_chain.dbs.events.max_open_conns")
 	conf.DbsEvents.ConnMaxLifetime = viper.GetDuration("server_chain.dbs.events.conn_max_lifetime")
+
+	conf.MaxChallengeCompletionTime = viper.GetDuration("smart_contracts.storagesc.max_challenge_completion_time")
```

Comment:
This MaxChallengeComplettionTime is not being used, the one that is being used is in the `config.SmartContractConfig`, please remove this one to avoid ambiguous.

Reply:
fixed